src/Sylius/Bundle/AddressingBundle/.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: php
 
+sudo: false
```

Comment:
No, it's not, see i.e. OrderBundle tests: https://travis-ci.org/Sylius/SyliusOrderBundle/jobs/102929723